Purchases.xcodeproj/project.pbxproj

```diff
@@ -2588,15 +2588,15 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		B3D5CFBB267282630056FA67 /* XCRemoteSwiftPackageReference "Nimble" */ = {
+		B3D5CFBB267282630056FA67 /* XCRemoteSwiftPackageReference "Nimble.git" */ = {
```

Comment:
not sure what's the update here, probably just some automated xcode thing

Reply:
i was seeing issues with nimble missing on my branch, maybe this will fix them!